addons/crm/models/crm_lead.py

```diff
@@ -1492,6 +1497,13 @@ def _pls_get_lead_pls_values(self, batch_mode=False):
         """
         leads_values_dict = OrderedDict()
         fields = ["stage_id", "team_id"] + self._pls_get_safe_fields()
+        query_tag_ids_1 = query_tag_ids_2 = ''
+        if 'tag_ids' in fields:
```

Comment:
maybe set a use_tag variable
like : `use_tag = bool(fields.pop(fields.index('tag_ids')) if 'tag_ids' in fields else False)` (looks a bit ugly..)
so that the tag_is is removed from field if it's in there.
Then use the `use_tag` directly above `query = sql.SQL... `as you don't use query_tag_ids_1 content anywhere else 
```
query_tag_select = ", t.id as tag_id" if use_tag else ""
query_tag_join = """LEFT JOIN crm_tag_rel rel ON l.id = rel.lead_id
                                LEFT JOIN crm_tag t ON rel.tag_id = t.id""" if use_tag else ""
query = sql.SQL(query % (query_tag_select, query_tag_join)).format(*args)
```

Reply:
done